src/transformers/default-schema-transformer.ts

```diff
@@ -147,6 +147,8 @@ function transformByRendering(question: FormField) {
     case 'group':
       handleLabOrders(question);
       break;
+    case 'date':
+      question.datePickerFormat = question.datePickerFormat ?? 'calendar';
```

Comment:
For backwards compatibility, do you want to add a case for "datetime" where we set `datePickerFormat` to "both" assuming "both" renders both the calendar and time picker?